projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -691,6 +695,27 @@ class="wpas-submit-<?php echo esc_attr( $connection_data['service_name'] ); ?>"
 		<?php if ( ! $all_done ) : ?>
 			<div id="pub-connection-tests"></div>
 		<?php endif; ?>
+
+		<?php if ( ! $all_healthy ) : ?>
+			<div class="error below-h2 publicize-token-refresh-message">
+				<?php
+					printf(
+						wp_kses(
+							/* translators: %s is the link to the connections page in Calypso */
+							__( 'Some of your social connections are broken! Reconnect them on the <a href="%s" rel="noopener noreferrer" target="_blank">connection management</a> page..', 'jetpack-publicize-pkg' ),
```

Comment:
Do you think we could get a count of the unhealthy connections? That way we could have another variation of this that says "One of your social..."

I wonder if we should not use an exclamation mark too. It should be a bit urgent, but we don't want to painc them

Reply:
Good idea I change this!